README.md

```diff
@@ -1,4 +1,27 @@
+## Introduction
 
+**What?** The [Neurodata Without Borders: Neurophysiology][nwb-neurophysiology] (NWB:N) Hackathon is a week-long hands
+on activity in which neurophysiology researchers create solutions using the open source NWB:N software packages.
+Participants work collaboratively on solutions to develop a unified data format for cellular-based neurophysiology data, focused on the dynamics of groups of neurons measured under a large range of experimental conditions. In contrast to conferences and workshops where the primary focus is to report results, the objective of the Hackathon is to provide a venue for creators of neurophysiology open-source software to collaboratively work.
```

Comment:
Participants work collaboratively on solutions to develop a unified data format for cellular-based neurophysiology data, focused on the dynamics of groups of neurons measured under a large range of experimental conditions. 

=>

Participants work collaboratively on solutions that use the NWB unified data format for cellular-based neurophysiology data, which is focused on the dynamics of groups of neurons measured under a large range of experimental conditions.

Reply:
the objective of the Hackathon is to provide a venue for creators of neurophysiology open-source software to collaboratively work

=>

the objective of the Hackathon is to provide a venue for creators and users of neurophysiology open-source software to collaboratively work on any related research projects